### DISCUSSION TITLE
IPASIR strings for options

### DISCUSSION THREAD

**Discussion by m-fleury** [ipasir2/ipasir2#39] [Ideas]
Is there any way to set a filename as option? For example to produce proofs? 

This is for example relevant to produce IDRUP proofs (for incremental SAT solving, so you really want to use IPASIR2).

**Comment by fkutzner** [comment]
Hi Mathias, currently this is not supported in IPASIR2. I think if we add support for proof generation, we should add a dedicated function for setting the filename. I've updated and reopened issue #16.
